package/network/utils/wwan/files/wwan.sh

```diff
@@ -114,6 +117,12 @@ proto_wwan_teardown() {
 	comgt)		proto_3g_teardown $@ ;;
 	*cdc_ncm)	proto_ncm_teardown $@ ;;
 	esac
+
+	echo "Stopping rssi_led_script"
+	led_proc_id=$(ps | grep set_3g_rssi_led | awk '/start/ {print $1}')
+	echo "pid: $led_proc_id"
+	kill "$led_proc_id"
+	set_3g_rssi_led.sh stop
```

Comment:
Currently the script is unable to stop the set_3g_rssi_led script.

Manually I can get the proc id:
```
root@lede:/tmp# ps | grep set_3g_rssi_led | awk '/start/ {print $1}'
1165
```

but the script is unable as shown by logread:
```
Sun Jan 15 09:25:46 2017 daemon.notice netifd: wan3g (1336): Stopping rssi_led_script
Sun Jan 15 09:25:46 2017 daemon.notice netifd: wan3g (1336): pid:
Sun Jan 15 09:25:46 2017 daemon.notice netifd: Interface 'wan3g' is now down
```

Some idea to help me?